NineChronicles.Headless.Executable/Program.cs

```diff
@@ -231,16 +226,41 @@ static void ConfigureSentryOptions(SentryOptions o)
                 graphQLSecretTokenPath, noCors, nonblockRenderer, nonblockRendererQueue, strictRendering,
                 logActionRenders, isDev, blockInterval, reorgInterval, confirmations,
                 txLifeTime, messageTimeout, tipTimeout, demandBuffer, staticPeerStrings, skipPreload,
-                minimumBroadcastTarget, bucketSize, chainTipStaleBehaviorType, txQuotaPerSigner, maximumPollPeers
+                minimumBroadcastTarget, bucketSize, chainTipStaleBehaviorType, txQuotaPerSigner, maximumPollPeers,
+                sentryDsn, sentryTraceSampleRate
             );
 
-#if SENTRY || !DEBUG
+#if SENTRY || ! DEBUG
             loggerConf = loggerConf
                 .WriteTo.Sentry(o =>
                 {
                     o.InitializeSdk = false;
                 });
+
+            using var _ = SentrySdk.Init(o =>
+            {
+                o.SendDefaultPii = true;
+                o.Dsn = headlessConfig.SentryDsn;
+                // TODO: o.Release 설정하면 좋을 것 같은데 빌드 버전 체계가 아직 없어서 어떻게 해야 할 지...
+                // https://docs.sentry.io/workflow/releases/?platform=csharp
+                //o.Debug = true;
```

Comment:
```suggestion
                // TODO: We need to specify `o.Release` after deciding the version scheme.
                // https://docs.sentry.io/workflow/releases/?platform=csharp
                //o.Debug = true;
```

Reply:
I think that the informational version seems helpful even now. (see also: https://github.com/planetarium/NineChronicles.Headless/pull/1700/commits/c3ca8f1840a26fef49754461f79850becc18a74b#diff-22f31cb0fcc7801c787cc20e29afeee1c164b0979787ac9c1245cfbed40a3461R23)